flow/packet-headers/ipv4.yaml

```diff
@@ -112,15 +112,15 @@ components:
               Source address
```

Comment:
Should we add some clarification on when auto_dhcp can be used i.e. only when the traffic source is on top of a DHCP client interface ? [ Same for dst ? ]